consensus/service/src/byzantine_ledger.rs

```diff
@@ -441,15 +475,28 @@ impl<
                     .attempt_ledger_sync(&self.network_state, blocks_per_attempt)
                 {
                     log::error!(self.logger, "Could not sync ledger: {:?}", err);
-                    thread::sleep(Duration::from_secs(1));
+
+                    // The next time we attempt to sync is an exponential back-off based on how many
+                    // attempts we've done so far, capped at 60 seconds.
+                    let next_sync_at = now + Duration::from_secs(min(num_sync_attempts + 1, 60));
```

Comment:
This looks like linear (as opposed to exponential) backoff, but that works, too.

Reply:
Yup, that's correct. I changed the comment.